docs/extend/extend-apiml/api-mediation-oidc-authentication.md

```diff
@@ -164,4 +146,14 @@ The OIDC provider returns an HTTP 40x error code.
 The client application is not properly configured in the API ML Gateway.
 
 **Solution**  
-Check that the `client_id` and `client_secret` configured in the API ML Gateway correspond to the `client_id` and `client_secret` of the client application as configured in the OIDC provider.
+Check that the URL `jwks_uri` contain actual key for the OIDC token validation.
+
+
+## Tips
+API ML Gateway exposes validate token `POST /gateway/api/v1/auth/oidc-token/validate` operation which is suitable during the OIDC setup. It expects JSON `{ token, serviceId }`. Call allows to verify that OIDC token is trusted by the API ML. Accounts mapping step is not included in that flow. 
```

Comment:
What is the "flow" in this context?

Reply:
OIDC is handled via different flows through the applications and involved services. 

This statement means that if you call /gateway/api/v1/auth/oidc-token/validate endpoint the Gateway service won't do the mapping request to the ESM.